utilities/tools/helper/ConvertTo-OrderedHashtable.ps1

```diff
@@ -87,13 +87,13 @@ function ConvertTo-OrderedHashtable {
                 }
 
                 # Case: Array of objects
-                $hashTableElements = $JSONObject[$currentLevelKey] | Where-Object { $_.GetType().Name -eq 'Hashtable' }
+                $hashTableElements = $JSONObject[$currentLevelKey] | Where-Object { $_.GetType().Name -in @('Hashtable', 'OrderedHashtable') }
```

Comment:
Should we merge with #2294 ?

Reply:
It serves a different purpose, but sure. Let me change the target branch